instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -1415,36 +1418,73 @@ def invoke_model_tool_call(
     new_stream = io.BytesIO(response_0_raw_body)
     response_0["body"] = StreamingBody(new_stream, len(response_0_raw_body))
 
-    tool_requests_ids = [
-        content["id"]
-        for content in response_0_body["content"]
-        if content["type"] == "tool_use"
-    ]
+    if "anthropic.claude" in llm_model_value:
+        tool_requests_ids = [
+            content["id"]
+            for content in response_0_body["content"]
+            if content["type"] == "tool_use"
+        ]
+    else:
+        tool_requests_ids = [
+            content["toolUse"]["toolUseId"]
+            for content in response_0_body["output"]["message"]["content"]
+            if "toolUse" in content
+        ]
+
     assert len(tool_requests_ids) == 2
-    tool_call_result = {
-        "role": "user",
-        "content": [
-            {
-                "type": "tool_result",
-                "tool_use_id": tool_requests_ids[0],
-                "content": "50 degrees and raining",
-            },
-            {
-                "type": "tool_result",
-                "tool_use_id": tool_requests_ids[1],
-                "content": "70 degrees and sunny",
-            },
-        ],
-    }
 
-    # remove extra attributes from response
-    response_0_body.pop("id")
-    response_0_body.pop("stop_reason")
-    response_0_body.pop("stop_sequence")
-    response_0_body.pop("usage")
-    response_0_body.pop("model")
-    response_0_body.pop("type")
-    messages.append(response_0_body)
+    if "anthropic.claude" in llm_model_value:
+        # remove extra attributes from response
```

Comment:
why does the test need to do this though?

Reply:
Indeed it would be cleaner to just create a new message with the proper role and content instead of this.